### PR TITLE
install.sh : Add checks to remove old config versions

### PR DESCRIPTION
**Pull request by AdityaGarg8**

### CODE REVIEW

**Comment by kekrby** [comment]
I think it makes more sense to have a separate `uninstall-old.sh` script.

**Comment by kekrby** [comment]
I think it is a good idea to ask for confirmation and I'd rather have the script not delete the user's pulseaudio configuration.

**Comment by kekrby** [comment]
While checking the model makes it a bit better, I think it is better just to print a warning as the user might have other settings saved instead of deleting their `pulseaudio` configuration.

**Comment by AdityaGarg8** [comment]
> While checking the model makes it a bit better, I think it is better just to print a warning as the user might have other settings saved instead of deleting their `pulseaudio` configuration.

I didn't exactly get what you mean here. Though, could you do this minor edit?

File: install.sh [hunk 1]
```diff
@@ -8,6 +8,41 @@ then
...
+done
+
+config_home=${XDG_CONFIG_HOME:-~/.config}
+
+for file in "$config_home/pulse/default.pa" \
+            "$config_home/pulse/daemon.conf"
+do
+    if [ -f "$file" ]
+    then
+        echo "You seem to have pulseaudio configuration files present, don't forget to update them if they contain anything specific to T2 Macs"
```
Comment (AdityaGarg8) [inline install.sh:46]: Well I guess it shall be better to ask the user to delete them or not?

Comment (kekrby) [reply install.sh:46]: In my opinion the user should review the files to see if they can be deleted as they might have put other settings there and have forgotten about it.

**Comment by kekrby** [comment]
I don't think the last commit is needed, the script already uses the correct directory.

**Pull request merged**